test.tsx

```diff
@@ -56,6 +74,30 @@ describe('react2angular', () => {
     expect(TestAngularOne.controller).not.toBe(undefined)
   })
 
+  it('should use the propTypes when present and no bindingNames were specified', () => {
+    const reactAngularComponent = react2angular(TestThreeWithPropTypes)
+
+    expect(reactAngularComponent.bindings).toEqual({
+      bar: '<',
+      baz: '<',
+      foo: '<'
+    })
+  })
+
+  it('should use the bindingNames when present over the propTypes', () => {
+    const reactAngularComponent = react2angular(TestThreeWithPropTypes, ['foo'])
+
+    expect(reactAngularComponent.bindings).toEqual({
+      foo: '<'
+    })
+  })
+
+  it('should have empty bindings when parameter is an empty array', () => {
+    const reactAngularComponent = react2angular(TestThreeWithPropTypes, [])
+
+    expect(reactAngularComponent.bindings).toEqual({})
+  })
```

Comment:
If you're up for it, perhaps add a test for calling `react2angular` with no 2nd argument.